noise-research/secure-transfer.md

```diff
@@ -0,0 +1,243 @@
+# Secure Transfers with Noise
+
+In this document we describe a compound protocol to enable two devices to mutually authenticate and securely exchange (arbitrary) information. 
+It consists of two main subprotocols or *phases*:
+
+- [Device Pairing](#Device-Pairing): the devices exchange and authenticate their long term device ID static keys;
+- [Secure Transfer](#Secure-Transfer): the devices securely exchange in encrypted form information using key material obtained during a successful pairing phase.
+
+## Device Pairing
+
+In the pairing phase, a device `B` requests to be paired to a device `A`. Once the two devices are paired, the devices will be mutually authenticated and will share a Noise session within which they can securely exchange information.
+
+The request is made by exposing a QR code that, by default, has to be scanned by device `A`. 
+If device `A` doesn't have a camera while device `B` does, [it is possible](#Rationale) to execute a slightly different pairing (with same security guarantees), where `A` is exposing a QR code instead.
+
+### Employed Cryptographic Primitives
+
+- `H`: the underlying hash function, i.e. SHA-256;
+- `HKDF`: a key derivation function (based on SHA-256);
+- `Curve25519`: the underlying elliptic curve for Diffie-Hellman (DH) operations.
+
+### The `WakuPairing` Noise Handshake
+
+The devices execute a custom handshake derived from `X1X1`, where they mutually exchange and authenticate their device static keys by exchanging messages over the content topic
```

Comment:
If I understood correctly, the paring happens by scanning QR codes, right? then why we need a contentTopic? it is not over a p2p network?